src/subscriptions.jl

```diff
@@ -93,11 +93,11 @@ function open_subscription(fn::Function,
         "payload" => payload
     )
     message_str = JSON3.write(message)
-
-    HTTP.WebSockets.open(client.ws_endpoint; retry=retry, headers=client.headers) do ws
+    throw_if_assigned = Ref{GraphQLError}()
+    HTTP.WebSockets.open(client.ws_endpoint; retry=retry, headers=client.headers, suppress_close_error=false) do ws
```

Comment:
What does `suppress_close_error` do here? Struggling to find any docs for it! Also it looks like the [default value ](https://github.com/JuliaWeb/HTTP.jl/blob/ec17649a49c078de5d173837080f38701d137c1c/src/WebSockets.jl#L350)is `false` already?